### PR TITLE
Added Swift Package Manager support

### PR DESCRIPTION
**Pull request by treatwell-marius**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,23 @@
...
+
+or add dependency to your `Package.swift`
+
+```swift
+import PackageDescription
+
+let package = Package(
+    name: "MySPMLibrary",
+    dependencies: [
+        .package(url: "https://github.com/treatwell/twuitests.git", .upToNextMajor(from: "0.2.0"))
```
Comment (davdroman) [inline README.md]: You can remove `.upToNextMajor(` and use just `from: "0.2.0"`. Means the same.

Comment (treatwell-marius) [reply README.md]: 👍

File: Example/Example/Base.lproj/LaunchScreen.storyboard [hunk 2]
```diff
@@ -13,7 +13,7 @@
                     <view key="view" contentMode="scaleToFill" id="Ze5-6b-2t3">
                         <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
-                        <color key="backgroundColor" red="1" green="1" blue="1" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+                        <color key="backgroundColor" xcode11CocoaTouchSystemColor="systemBackgroundColor" cocoaTouchSystemColor="whiteColor"/>
```
Comment (lokatorius) [inline Example/Example/Base.lproj/LaunchScreen.storyboard]: xcode11CocoaTouchSystemColor ? :D

**Review by lokatorius — APPROVED** [review]
nice work!

File: Example/Example/Base.lproj/LaunchScreen.storyboard (hunk 2 above)
Comment (treatwell-marius) [reply Example/Example/Base.lproj/LaunchScreen.storyboard]: 🤷‍♂️

File: README.md (hunk 1 above)
Comment (davdroman) [reply README.md]: Is the change from 2 to 20 intentional? @treatwell-marius

Comment (treatwell-marius) [reply README.md]: Yes 😄 I am going to bump version to `0.20.0`, current one is `0.17`

**Review by davdroman — APPROVED** [review]

**Pull request merged**